src/components/Card/card.module.scss

```diff
@@ -0,0 +1,130 @@
+.card {
+    --bg-color: white;
+
+    box-shadow: $shadow-object-m;
+    background-color: var(--bg-color);
+    border-radius: $corner-radius-xl;
+    color: black;
+    display: block;
+    font-family: $octuple-font-family;
+    gap: $space-m;
+    padding: $space-ml;
+    width: 360px;
+    position: relative;
+
+    &.carousel {
```

Comment:
this is the paginated version as seen in our mocks? no way to view this in the storybook